docs/developers-csi-manila.md

```diff
@@ -1,3 +1,17 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
```

Comment:
is this `*generated with [DocToc](https://github.com/thlorenz/doctoc)*` is autogenerated? can we remove it/or add it at the end of contents?

Reply:
i think it's autogenerated and used by DocToc to identify the ToC section, @jichenjc is that correct?

Reply:
yes, it's auto generated , as it doesn't impact our doc, so I keep them